src/gui/mainclient.cpp

```diff
@@ -202,8 +178,5 @@ int main(int argc, char **argv) {
         }
     }
 
-    // Make sure everything flushes
-    auto sentryClose = qScopeGuard([] { sentry_close(); });
```

Comment:
sentry_close is not done anymore?

Reply:
You are right, the call to sentry_close  in SentryHandler destructor get lost on the way. Fix it here: [00d56f6](https://github.com/Infomaniak/desktop-kDrive/pull/274/commits/00d56f60ed4d5bb9d49708abe5a02ea33c927330)